package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "device-id-widget",
+  "widgetName": "DeviceIdWidget",
+  "version": "1.0.0",
+  "description": "",
+  "copyright": "Mendix BV",
+  "scripts": {
+    "pretest": "tsc",
+    "test": "karma start --single-run --codeCoverage && node tests/remap.js",
+    "lint": "tslint -c tslint.json '**/*.ts' '**/*.tsx' --exclude '**/node_modules/**'"
+  },
+  "pre-commit": [
+    "pretest",
+    "lint",
+    "test"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendixlabs/device-id-widget"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/mendixlabs/device-id-widget/issues"
+  },
+  "devDependencies": {
+    "@types/big.js": "0.0.31",
```

Comment:
I think this library is required by mendix. It is in all our projects. Let me trying removing it and i see. cc @edwinmp